sdk/avs/Azure.ResourceManager.Avs/api/Azure.ResourceManager.Avs.netstandard2.0.cs

```diff
@@ -1136,55 +1168,49 @@ public partial class DiskPoolVolume
+    public readonly partial struct DnsServiceStatus : System.IEquatable<Azure.ResourceManager.Avs.Models.DnsServiceStatus>
     {
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
-        public DnsServiceStatusEnum(string value) { throw null; }
-        public static Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum Failure { get { throw null; } }
-        public static Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum Success { get { throw null; } }
-        public bool Equals(Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum other) { throw null; }
+        public DnsServiceStatus(string value) { throw null; }
+        public static Azure.ResourceManager.Avs.Models.DnsServiceStatus Failure { get { throw null; } }
+        public static Azure.ResourceManager.Avs.Models.DnsServiceStatus Success { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.Avs.Models.DnsServiceStatus other) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override bool Equals(object obj) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
         public override int GetHashCode() { throw null; }
-        public static bool operator ==(Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum left, Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum right) { throw null; }
-        public static implicit operator Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum (string value) { throw null; }
-        public static bool operator !=(Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum left, Azure.ResourceManager.Avs.Models.DnsServiceStatusEnum right) { throw null; }
+        public static bool operator ==(Azure.ResourceManager.Avs.Models.DnsServiceStatus left, Azure.ResourceManager.Avs.Models.DnsServiceStatus right) { throw null; }
+        public static implicit operator Azure.ResourceManager.Avs.Models.DnsServiceStatus (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.Avs.Models.DnsServiceStatus left, Azure.ResourceManager.Avs.Models.DnsServiceStatus right) { throw null; }
         public override string ToString() { throw null; }
     }
-    public partial class Encryption
-    {
-        public Encryption() { }
-        public Azure.ResourceManager.Avs.Models.EncryptionKeyVaultProperties KeyVaultProperties { get { throw null; } set { } }
-        public Azure.ResourceManager.Avs.Models.EncryptionState? Status { get { throw null; } set { } }
-    }
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
     public readonly partial struct EncryptionKeyStatus : System.IEquatable<Azure.ResourceManager.Avs.Models.EncryptionKeyStatus>
```

Comment:
This might also need a prefix - other RPs have something very similar to this

Reply:
Prepend the RP prefix.